backend/dal/Services/Admin/Concrete/ParcelService.cs

```diff
@@ -70,6 +69,18 @@ public Paged<Parcel> Get(int page, int quantity, ParcelFilter filter = null)
             if (!String.IsNullOrWhiteSpace(filter.Address))
                 query = query.Where(p => EF.Functions.Like(p.Address.Address1, $"%{filter.Address}%") || EF.Functions.Like(p.Address.City.Name, $"%{filter.Address}%"));
 
+            if (!String.IsNullOrWhiteSpace(filter.Description))
```

Comment:
We can now search for `Description, Municipality, Zoning, ZoningPotential`

Reply:
The following do "start-with" searches. Wouldn't it make more sense to switch to a "contains" search - especially for the `Description` field. I think all should be switched, thoughts?

Reply:
As a user I generally expect searches to use contains instead of startswith.

Reply:
I had chosen `starts-with` for the performance advantage and thought it actually made more sense for most of these filters (Municipality, Zoning, ZoningPotential), but it should be a `contains` for Description.  I'll probably just make them all `contains` for now for simplicity.